.sonarcloud.properties

```diff
@@ -4,7 +4,7 @@
 
 # Path to sources
 sonar.sources=.
-sonar.exclusions=**/localization/**/index.ts
+sonar.exclusions=**/localization/**/index.ts,packages/legacy/core/cli.js
```

Comment:
Should this go in scripts?

Reply:
I think it could. It's Clecio's tool, I'll bring it up in standup tomorrow and see if he thinks that's a good idea